test/byContract/IexecPoco/01_initialize.test.ts

```diff
@@ -107,18 +108,12 @@ describe('Poco', async () => {
                 volume: 9,
             };
             // Request order is matched in 2 deals
-            const { dealId: dealId0 } = await iexecWrapper.signAndMatchOrders({
-                app: appOrder,
-                dataset: datasetOrder,
-                workerpool: workerpoolOrder0,
-                requester: requestOrder,
-            });
-            const { dealId: dealId1 } = await iexecWrapper.signAndMatchOrders({
-                app: appOrder,
-                dataset: datasetOrder,
-                workerpool: workerpoolOrder1,
-                requester: requestOrder,
-            });
+            const { dealId: dealId0 } = await iexecWrapper.signAndMatchOrders(
+                new IexecOrders(appOrder, datasetOrder, workerpoolOrder0, requestOrder),
```

Comment:
```suggestion
                appOrder, datasetOrder, workerpoolOrder0, requestOrder,
```
Could you please update `signAndMatchOrders` to match this function signature (hence in required places `orders.toArray()`)?

Reply:
Yes [Update signAndMatchOrders signature](https://github.com/iExecBlockchainComputing/PoCo/pull/123/commits/54ec4a04d817d0a48f15899d6733c6ed6d45ae6e)

Reply:
Thanks :+1: